### ISSUE TITLE
Tap view fails and/or hangs to simulator when runnin on CI

### ISSUE THREAD

**Issue by zaxy78** [kif-framework/KIF#961]
I have few tests running great via Xcode.

When I use our CI system (fastlane scan via terminal/bot)  it gets stuck.
Simulator usually hangs/gets stuck. 

When logging I can see the waitForView was successful, the view was found, but when I tap it, it gets stuck. No error. No crash. simply stuck (infinite wait/loop somehow??)

I can't debug it since it won't reproduce in Xcode. only when runing via CI.

Any Idea?

**Comment by zaxy78** [comment]
I found a workaround but I'm not sure why it worked.

I decide it just loads slowly, and I added a wait(with TimeInterval (5 sec)) in the BeforeAll of my TestSuite, and it works. i can even speed up to 75 (animation time) and it runs great.

Why is this?
is it because _our app_ loads "slowly", and doing background work while Main UI loads? 
why does it even change the tapView() implementation to go into a loop/lock mode?
